### PR TITLE
Adds chakra theme to storybook with Button component as an example

### PR DESCRIPTION
**Pull request by georgeweiler**

### CODE REVIEW

**Comment by georgeweiler** [comment]
I'm open to discussion about how to better manage our storybook. This PR is mostly for the config and to get us chatting. There may be other examples of how we've used storybook in past projects that we should adopt.

File: .github/workflows/chromatic.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# .github/workflows/chromatic.yml
+
+# Workflow name
+name: "Chromatic"
+
+# Event for the workflow
+on: push
+
+# List of jobs
+jobs:
```
Comment (nkuba) [inline .github/workflows/chromatic.yml:10]: Can we remove the comments?

```suggestion
name: "Chromatic"

on: push

jobs:
```

File: .github/workflows/chromatic.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  chromatic-deployment:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
```
Comment (nkuba) [inline .github/workflows/chromatic.yml:14]: There is a newer version available.

```suggestion
      - uses: actions/checkout@v2
```

File: .github/workflows/chromatic.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      - name: Install dependencies
+        run: yarn
+      - name: Publish to Chromatic
+        if: github.ref != 'refs/heads/main'
+        uses: chromaui/action@v1
+        with:
+          token: ${{ secrets.CI_GITHUB_TOKEN }}
+          # Should set this up from a secret: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
```
Comment (nkuba) [inline .github/workflows/chromatic.yml:22]: I configured the `CHROMATIC_PROJECT_TOKEN` secret for this repository, you're good to use it.

File: .github/workflows/chromatic.yml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+          projectToken: adb814d4adb6
+      - name: Publish to Chromatic and auto accept changes
+        if: github.ref == 'refs/heads/main'
+        uses: chromaui/action@v1
+        with:
+          token: ${{ secrets.CI_GITHUB_TOKEN }}
+          projectToken: adb814d4adb6
+          autoAcceptChanges: true
```
Comment (nkuba) [inline .github/workflows/chromatic.yml:30]: If `autoAcceptChanges` is the only difference in these two steps we could combine them:

```suggestion
      - name: Publish to Chromatic and auto accept changes
        if: 
        uses: chromaui/action@v1
        with:
          token: ${{ secrets.CI_GITHUB_TOKEN }}
          projectToken: adb814d4adb6
          # Auto accept changes for main branch commits.
          autoAcceptChanges: ${{ github.ref == 'refs/heads/main' }}
```

**Review by nkuba — CHANGES_REQUESTED** [review]
I reviewed the workflow only and left some minor comments.

**Comment by georgeweiler** [comment]
Great suggestions @nkuba! I've pushed some updates. I had to make some additional changes to support v2: https://www.chromatic.com/docs/github-actions#support-for-codeactionscheckoutv2code

**Comment by georgeweiler** [comment]
I'm going to merge this into the chakra-theming branch so we are only working with 1 pull request for our theme & storybook

**Pull request merged**